packages/blocks/src/components/format-quick-bar/format-bar-node.ts

```diff
@@ -210,6 +217,41 @@ export class FormatQuickBar extends WithDisposable(LitElement) {
     </div>`;
   }
 
+  private renderDatabaseAction() {
```

Comment:
I think using a functional component here could be better. How about migrating the `_paragraphPanelTemplate` into to the approach similar with the `block-hub.ts` @lawvs ?